index.html

```diff
@@ -195,10 +195,27 @@ <h4 class="boxConnect2">LET’S CONNECT</h4>
             <ul><img src="images/Frame 9 (1).png" alt="Skills"></ul>
         </div>
     </section>
-    <section>
-        
+    
+    <section class="contactMe">
+        <h1>Contact me</h1>
+        <p>If you have an application you are interested in developing, a feature that you need built or a project
+            that needs coding.
+            I’d love to help with it!</p>
+        <div class="form">
+            <form>
+                <div class="form-group">
+                    <input type="text" placeholder="Name" maxlength="30">
+                </div>
+                <div class="form-group">
+                    <input type="email" placeholder="Email">
+                </div>
+                <div class="form-group">
+                    <textarea cols="30" rows="10" maxlength="500" placeholder="Write your message here..."></textarea>
+                </div>
```

Comment:
- [ ] I would kindly suggest you add the `required` attribute to all the input tag as this will give the form a validation and avoid the user to submit an empty form, also this is part of the project requirement.